spacy/lang/nn/tokenizer_exceptions.py

```diff
@@ -0,0 +1,227 @@
+from ...symbols import NORM, ORTH
+from ...util import update_exc
+from ..tokenizer_exceptions import BASE_EXCEPTIONS
+
+_exc = {}
+
+
+for exc_data in [
+    {ORTH: "jan.", NORM: "januar"},
+    {ORTH: "feb.", NORM: "februar"},
+    {ORTH: "mar.", NORM: "mars"},
+    {ORTH: "apr.", NORM: "april"},
+    {ORTH: "jun.", NORM: "juni"},
```

Comment:
Is "juli" missing on purpose?

Reply:
Yes. The abbreviation "jul(.)" for "juli" also means "Christmas" i Norwegian Nynorsk.